guidelines/environments.md

```diff
@@ -72,33 +72,48 @@ Use any of the 4 sets of Client/AccessTokens to access the Net Pricing demo ente
+* **AccessToken** - Unique token per enterprise. Can be provided to you by the enterprise admin.
 
 ### Request limits
 
 * 3000 requests per `AccessToken` within 15 minutes
 * 1500 requests per endpoint per `AccessToken` within 15 minutes
 
-> **NOTE** All requests utilizing [Pagination](guidelines/pagination.md) have their limits increased by 10 times.
+> **Pagination**: All requests utilizing [Pagination](pagination.md) have their limits increased by 10 times.
 
 ## Taxations
 
-Each enterprise is located in a specific [Tax environment](../operations/taxenvironments.md#tax-environment) that offers a list of applicable [Taxations](../operations/taxations.md#taxation). The numeric value of the taxations are represented by [Tax rate codes](../operations/taxations.md#tax-rate) that are accepted within the tax environment. 
+Each enterprise operates within a specific [Tax environment](../operations/taxenvironments.md#tax-environment), which defines the applicable [Taxations](../operations/taxations.md#taxation). The tax rates are represented by [Tax rate](../operations/taxations.md#tax-rate) codes accepted within that environment.
+Instead of using numeric tax rates like `0.1`, use [Tax rate](../operations/taxations.md#tax-rate) codes such as `AT-S` to accurately represent and calculate the correct taxation for each accounting item in Mews.
 
-Instead of numeric tax rates such as `0.1`, use [Tax rate codes](../operations/taxations.md#tax-rate) such as `AT-S` in order to represent and calculate the correct taxation for each accounting item in Mews. To obtain the applicable codes, first download the enterprise information with [Get configuration](../operations/configuration.md#get-configuration) to identify the Tax Environment, then filter for the applicable [Tax rate codes](../operations/taxations.md#tax-rate) from all tax environment information retrieved via [Get all tax environments](../operations/taxenvironments.md#get-all-tax-environments). 
+To obtain the applicable codes:
 
-Make sure to note the validity intervals as well as any government announcements to anticipate changes to tax rates. Should any changes occur, re-retrieve enterprise and tax environment information to identify the new tax rate codes.
+1. Download the enterprise information using [Get configuration](../operations/configuration.md#get-configuration) to identify the Tax Environment.
+2. Filter for the applicable [Tax rate](../operations/taxations.md#tax-rate) codes from the tax environment information retrieved via [Get all tax environments](../operations/taxenvironments.md#get-all-tax-environments).
+3. Be sure to note the validity intervals, and monitor any government announcements for changes to tax rates. If changes occur, re-retrieve the enterprise and tax environment information to identify the new tax rate codes.
 
-## Whitelisting
+## Allowlisting
```

Comment:
suggestion: While you are editing this section, you could rename it to "IP address allowlisting" since it's not immediately clear what you're allowlisting.